src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Ball.Color.values().length);
+        return Ball.Color.values()[index].toString();
```

Comment:
Instead of using `toString()`, you should use the `name()` method to get the String representation of the enum constants as per the checklist. This ensures that you get the exact name of the enum constant even if the `toString()` method is overridden.